index.html

```diff
@@ -182,5 +185,18 @@ <h3>LET'S CONNECT</h3>
                     </ul>
 
                 </section>
+                <section class="contactform1">
+                    <h1>Contact me</h1>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a 
+                        project that needs coding. I'd love to help with it </p>
+                        <nav>
+                        <form action="https://formspree.io/f/xrgjovwd" method="post">
+                            <input type="text" id="name" placeholder="Yeramias" maxlength="30" required>
+                            <input type="email" id="mail" name="email address" placeholder="natajayarijl4@gmail.com" required>
+                            <textarea id="msg" rows="5" name="user-message" placeholder="write your message here" maxlength="500" required></textarea>
+                            <button type="submit">Get It Touch </button>
+                        </form>
```

Comment:
- [ ] Kindly make sure to use flexbox to style your form elements, this is a project requirement.